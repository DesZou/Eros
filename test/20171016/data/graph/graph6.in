50 250 62
20 9 342
25 27 712
13 1 23
11 48 654
37 34 707
4 19 944
6 11 943
8 3 2
29 48 119
2 10 426
16 14 604
45 36 348
30 35 188
1 2 826
19 13 909
34 9 955
2 13 523
31 45 10
36 48 305
11 45 405
22 13 983
21 29 913
50 33 345
49 32 256
49 46 704
36 30 343
36 38 309
19 35 742
42 7 952
19 5 922
13 3 656
17 1 46
27 40 952
32 31 945
4 24 939
33 16 546
9 47 703
44 24 985
17 31 822
12 14 781
36 11 940
49 1 28
25 37 431
35 17 571
34 40 935
31 40 492
25 44 4
1 9 805
10 1 718
23 18 161
40 25 219
34 17 45
47 33 956
30 11 233
40 49 41
22 42 366
35 22 709
44 20 849
14 50 435
10 17 593
47 31 335
21 9 770
42 33 456
45 43 742
14 5 423
8 48 680
1 47 665
11 43 512
10 5 497
16 24 538
45 21 34
5 7 270
15 16 65
41 31 958
43 31 419
27 23 665
14 22 306
18 29 415
1 22 820
16 20 913
25 35 135
34 14 162
5 39 212
36 34 993
6 15 416
31 35 346
26 2 718
5 43 696
13 25 594
4 23 989
18 38 76
40 21 174
47 26 384
41 49 402
1 24 736
24 25 41
2 6 136
41 34 673
6 47 963
26 29 308
6 29 339
15 43 899
12 30 802
43 39 319
17 4 398
46 7 621
24 3 339
29 46 575
32 16 733
10 37 431
29 4 107
33 44 136
50 45 198
44 28 888
50 37 607
18 4 983
15 7 163
21 24 796
26 49 790
46 40 166
7 12 470
20 6 114
30 50 196
18 44 5
6 9 669
42 37 202
11 42 339
37 13 10
13 35 201
50 47 329
12 41 184
22 34 877
12 11 260
23 15 580
23 41 332
30 25 191
12 23 202
23 11 376
31 48 869
1 28 365
33 5 604
39 10 23
49 28 961
48 25 365
14 1 48
47 40 230
46 21 897
49 25 343
2 36 698
42 39 994
43 2 667
7 24 565
2 5 370
21 31 925
36 24 712
49 23 497
19 29 821
49 24 116
36 3 524
25 32 47
32 19 21
44 38 34
26 21 494
48 50 545
22 31 339
12 3 956
45 35 295
34 13 390
30 15 87
40 15 85
25 45 231
6 39 645
23 34 129
40 35 493
1 46 244
11 4 528
14 24 606
18 13 777
10 20 314
8 37 325
28 4 17
12 25 89
43 27 631
4 48 31
32 36 42
7 16 977
1 18 989
15 13 258
42 49 41
11 22 37
5 23 150
13 5 223
12 18 817
26 40 571
19 20 519
8 31 539
36 33 265
30 44 573
8 14 429
27 46 391
23 31 255
49 8 432
9 39 720
23 9 586
22 4 480
19 47 132
24 26 690
21 49 818
7 47 312
21 6 437
24 32 615
41 29 280
21 32 451
11 24 946
50 8 699
5 34 703
37 38 716
11 40 310
21 22 684
12 5 409
15 8 461
13 43 297
11 20 76
3 2 193
50 5 357
10 26 349
39 28 205
9 29 192
40 24 705
36 7 545
46 26 799
13 29 967
23 36 498
40 43 512
39 2 546
49 37 83
30 3 631
35 9 993
32 37 59
17 33 375
22 44 656
35 46 886
16 3 718
9 46 273
5 24 415
4 41 916
38 16 718
38 7 794
30 19 821
35 50 997
50
46 21
15 30
22 47
15 14
46 6
25 26
31 23
34 16
20 7
48 25
5 12
33 5
28 6
31 44
6 33
44 48
2 26
49 7
12 19
12 15
22 17
38 42
7 38
19 16
10 13
19 36
24 26
38 16
26 43
33 20
35 2
7 43
28 44
30 45
31 42
27 45
7 48
48 20
33 32
40 33
49 9
39 2
6 22
33 14
38 37
3 8
21 34
31 29
42 27
30 47
