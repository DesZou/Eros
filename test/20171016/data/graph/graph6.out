853
700
810
680
1082
816
740
780
1087
712
738
684
1171
584
903
609
1277
821
496
598
787
462
702
634
877
389
956
604
1213
896
824
1069
649
662
665
1183
1035
1022
526
564
997
951
1151
597
376
932
711
957
1142
623
